src/recipes/blobs-migrate/index.ts

```diff
@@ -0,0 +1,107 @@
+import { getStore, listStores } from '@netlify/blobs'
+import inquirer from 'inquirer'
+import pMap from 'p-map'
+
+import BaseCommand from '../../commands/base-command.js'
+import { error, log } from '../../utils/command-helpers.js'
+
+export const description = 'Migrate legacy Netlify Blobs stores'
+
+const BLOB_OPS_CONCURRENCY = 5
+
+interface Options {
+  args: string[]
+  command: BaseCommand
+}
+
+export const run = async ({ args, command }: Options) => {
+  if (args.length !== 1) {
+    return error(`Usage: netlify recipes blobs-migrate <name of store>`)
+  }
+
+  const [storeName] = args
+  const { api, siteInfo } = command.netlify
+  const clientOptions = {
+    apiURL: `${api.scheme}://${api.host}`,
+    siteID: siteInfo?.id ?? '',
+    token: api.accessToken ?? '',
+  }
+
+  // The store we'll copy from.
+  const oldStore = getStore({
+    ...clientOptions,
+    name: `netlify-internal/legacy-namespace/${storeName}`,
+  })
+
+  // The store we'll write to.
+  const newStore = getStore({
+    ...clientOptions,
+    name: storeName,
+  })
+  const { blobs } = await oldStore.list()
```

Comment:
does this really return all blobs? no pagination?

Reply:
Yes, see https://docs.netlify.com/blobs/overview/#list.